### PR TITLE
Catch Frontend DatabaseError

### PR DESCRIPTION
**Pull request by yiyinglovecoding**
Frontend will not break when 
- Database not generated 
- Database return empty result

### CODE REVIEW

File: applications/rag/frontend/container/main.py [hunk 1]
```diff
@@ -140,20 +140,28 @@ def index():
...
+
+            # chunkify query & fetch matches
+            query_emb = transformer.encode(query_text).tolist()
+            query_request = "SELECT id, text, text_embedding, 1 - ('[" + ",".join(map(str, query_emb)) + "]' <=> text_embedding) AS cosine_similarity FROM " + TABLE_NAME + " ORDER BY cosine_similarity DESC LIMIT 5;"
+            query_results = conn.execute(sqlalchemy.text(query_request)).fetchall()
+            conn.commit()
+            log.info(f"printing matches:")
+            for row in query_results:
+                print(row)
+        except sqlalchemy.exc.DatabaseError as err:
```
Comment (imreddy13) [inline applications/rag/frontend/container/main.py:157]: I wonder if we should make this more generic - catch all other Exceptions too and return empty string?

Comment (yiyinglovecoding) [reply applications/rag/frontend/container/main.py:157]: done

**Review by imreddy13 — APPROVED** [review]

**Pull request merged**